src/mod_webdav.c

```diff
@@ -34,7 +34,7 @@
 #endif
 
 #if defined(HAVE_LIBXML_H) && defined(HAVE_SQLITE3_H) \
- && defined(HAVE_UUID) && defined(HAVE_UUID_UUID_H)
+ && defined(HAVE_LIBUUID) && defined(HAVE_UUID_UUID_H)
```

Comment:
configure.ac would need to be modified to define HAVE_LIBUUID, too, before making this change.

Reply:
The pkg-config file for libuuid will take care of it, IIRC.

Reply:
Your responses to my feedback do not give me confidence in your code.
You did not test this, and you are incorrect, at least on my system.

Reply:
It should build now

Reply:
Thank you.